modules/wallet/wallet.go

```diff
@@ -122,3 +143,34 @@ func (w *Wallet) SendCoins(amount types.Currency, dest types.UnlockHash) ([]type
 	}
 	return txnSet, nil
 }
+
+// SendSiafunds creates a transaction sending 'amount' to 'dest'. The transaction
+// is submitted to the transaction pool and is also returned.
+func (w *Wallet) SendSiafunds(amount types.Currency, dest types.UnlockHash) ([]types.Transaction, error) {
+	tpoolFee := types.NewCurrency64(10).Mul(types.SiacoinPrecision) // TODO: better fee algo.
+	output := types.SiafundOutput{
+		Value:      amount,
+		UnlockHash: dest,
+	}
+
+	txnBuilder := w.StartTransaction()
+	err := txnBuilder.FundSiacoins(tpoolFee)
+	if err != nil {
+		return nil, err
+	}
+	err = txnBuilder.FundSiafunds(amount)
+	if err != nil {
+		return nil, err
+	}
+	txnBuilder.AddMinerFee(tpoolFee)
+	txnBuilder.AddSiafundOutput(output)
+	txnSet, err := txnBuilder.Sign(true)
```

Comment:
you know...you could give the txn builder a sticky error, like the block explorer:
```go
txnBuilder := w.StartTransaction()
txnBuilder.FundSiacoins(tpoolFee)
txnBuilder.FundSiafunds(amount)
txnBuilder.AddMinerFee(tpoolFee)
txnBuilder.AddSiafundOutput(output)
txnSet, err := txnBuilder.Sign(true) // returns the error of the first operation that failed
```
put it on the "maybe later" list :P